syntax="proto3";
package csproto;

import "google/protobuf/any.proto";

message ClientRequest {
    int32 client_version = 1;
    google.protobuf.Any msg = 100;
}

message ServerResponse {
    int32 code = 1;
    string msg = 2;
    oneof data {
        google.protobuf.Any success_data = 100;
        google.protobuf.Any error_data = 101;
    }
}